### PR TITLE
Add link to Chicago Inside Out to Resources.md

### PR DESCRIPTION
**Pull request by jeancochrane**
This PR adds a link to [Chicago Inside Out](https://placesjournal.org/article/chicago-inside-out) to the Resources page of our handbook. I found this piece particularly helpful for explaining the different offices that comprise Cook County, how those offices interact, and the way that property taxes affect the services that each office is able to provide. It also gives an overview of the recent history of the Sheriff, the State's Attorney, the President, and the Assessor, with some good background on the most important political events of the past decade that have affected these offices.

### CODE REVIEW

File: Handbook/Resources.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Useful materials that give an overview of Cook County's property tax system and
 - [:exclamation: The Cook County Property Tax Extension Process: A Primer](https://www.civicfed.org/civic-federation/publications/cook-county-property-tax-extension-process-primer-levies-tax-caps-and-) - Incredibly useful guide to the intricacies of the entire Cook County tax system
 - [:star: Why Are My Property Taxes So High?](https://www.chicagobusiness.com/crains-forum-taxes/forum-why-are-my-property-taxes-so-high) - Neat interactive from Crain's that compares two very different homes that pay roughly the same amount of property tax
 - [:star: How Lower-Income Americans Get Cheated on Property Taxes - NYTimes](https://www.nytimes.com/2021/04/03/opinion/sunday/property-taxes-housing-assessment-inequality.html) - Concise and accessible overview of the nation-wide issue of inequitable assessments
+- [:star: Chicago Inside Out - Places Journal](https://placesjournal.org/article/chicago-inside-out) - Deep dive into the roles and recent histories of the offices that comprise Cook County government, with a particular focus on the impact of property taxes on county services
```
Comment (ccao-jardine) [inline Handbook/Resources.md:46]: This article does a great job making salient some key confusions (like the fact that even though Cook County sends and collects bills, most taxpayer dollars fund non-county agencies, like city or school districts). Let's please retitle this to Chicago Inside Out (2018) - Places Journal, remove the star, and move it lower down either above or below the Chicago Triad Assessment Report (2018).

File: Handbook/Resources.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ Useful materials that give an overview of Cook County's property tax system and
...
 - [Learn About Illinois' Property Tax System](https://www.youtube.com/watch?v=l9YVudOQuZ4) - Illinois Realtors’ video about property tax system and bills
 - [Chicago Triad Assessment Report (2018) - Civic Consulting Alliance](https://prodassets.cookcountyassessor.com/s3fs-public/reports/2018ChicagoTriadSRSvf.pdf) - Study of 2018 assessment quality in the Chicago triad
 - [2020 Cook County Tax Rate Report](https://www.cookcountyclerkil.gov/sites/default/files/publications/2020_Tax_Rate_Report_Web.pdf) - Clerk’s annual tax rate report, for tax year 2020
```
Comment (ccao-jardine) [inline Handbook/Resources.md:49]: While we're here, might as well update this to the [2021 Cook County Tax 
 Rate Report](https://www.cookcountyclerkil.gov/sites/default/files/pdfs/2021%20Tax%20Rate%20Report.pdf), and move it higher in the list above the 2018 articles.

**Review by ccao-jardine — CHANGES_REQUESTED** [review]

**Comment by dfsnow** [comment]
@ccao-jardine Took your suggestions + added two additional items: my Chi Hack presentation and the Treasurer's new primer. Ready for re-review!

**Review by ccao-jardine — APPROVED** [review]
Excellent.

**Pull request merged**